db/seeds.d/02-roles_list.rb

```diff
@@ -3,32 +3,38 @@
-                                    :create_external_variables, :edit_external_variables, :destroy_external_variables,
-                                    :view_external_parameters, :create_external_parameters, :edit_external_parameters,
-                                    :destroy_external_parameters, :view_facts, :view_hostgroups, :view_hosts, :view_smart_proxies_puppetca,
-                                    :view_smart_proxies_autosign, :create_hosts, :edit_hosts, :destroy_hosts,
-                                    :build_hosts, :view_media, :create_media, :edit_media, :destroy_media,
-                                    :view_models, :view_operatingsystems, :view_ptables, :view_puppetclasses,
-                                    :import_puppetclasses, :view_config_reports, :destroy_config_reports,
-                                    :view_smart_proxies, :edit_smart_proxies, :view_subnets, :edit_subnets,
-                                    :view_statistics, :view_usergroups, :create_usergroups, :edit_usergroups,
-                                    :destroy_usergroups, :view_users, :edit_users, :view_realms, :view_mail_notifications,
-                                    :view_params, :view_ssh_keys],
-      'Bookmarks manager'       => [:view_bookmarks, :create_bookmarks, :edit_bookmarks, :destroy_bookmarks]
+      {Role::MANAGER => {:permissions => base_manage_permissions + view_permissions + manage_organizations_permissions,
+                         :description => 'Role granting all available permissions. With this role, user is able to do everything that admin can except for changing settings.'},
+       Role::ORG_ADMIN => {:permissions => base_manage_permissions + view_permissions - [:view_organizations],
+                           :description => 'Role granting all permissions except for managing organizations. It can be used to delegate administration of specific organization to a user. In order to create such role, clone this role and assign desired organizations'},
+       'Edit partition tables' => {:permissions => [:view_ptables, :create_ptables, :edit_ptables, :destroy_ptables], :description => 'Role granting permissions required for managin partition tables'},
+       'View hosts' => {:permissions => [:view_hosts],
+                        :description => 'Role granting permission only to view hosts'},
+       'Edit hosts' => {:permissions => [:view_hosts, :edit_hosts, :create_hosts, :destroy_hosts, :build_hosts],
+                        :description => 'Role granting permissions to update hosts. For features provided by plugins, you might need to combine this role with roles provided by those plugins'},
+       Role::VIEWER => {:permissions => view_permissions, :description => 'Role granting read only access. Users with this role can see all data but can not do any modifications'},
+       'Site manager' => {:permissions => [:view_architectures, :view_audit_logs, :view_authenticators, :access_dashboard,
```

Comment:
The indentation  is off here and there are som inconsistent whitespaces. I wonder why rubocop doesnt scream on us?

Reply:
Brw I thought we us spaces after `{ ` in hashes

Reply:
I intentionally broke the aligning of the hash arrows, I don't think it's good since when you add a key that requires one more space, you need to change all other lines. Rubocop does not scream, since in Ruby this is accepted, this is not puppet.

It might seem a bit off since one key is string starting with `'`, the other is constant but that's aligned correctly.

I will add space after `{`, it's not consistent in the app but I generally prefer it.

Reply:
I haven't meant the puppet style, purely the spaces (thanks, that's one of the few cops I would like to see around), yes, the ' and constants caused them to look misaligned.